dockernet/config/evmos_ica.json

```diff
@@ -0,0 +1,24 @@
+{
+    "interchainaccounts": {
+        "host_genesis_state": {
+            "active_channels": [],
+            "interchain_accounts": [],
+            "port": "icahost",
+            "params": {
+                "host_enabled": true,
+                "allow_messages": [
```

Comment:
this seems to be a different set of messages compared to the ones we'll support

Reply:
updated